### PR TITLE
Prepare for v0.8

### PR DESCRIPTION
**Pull request by ludwigschwardt**
The first of the release PRs...

This updates copyright / licence and changelog.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -16,11 +16,15 @@
...
 ################################################################################
 
+import os
+
 from setuptools import setup, find_packages
 
 
-with open('README.rst') as readme:
-    long_description = readme.read()
+here = os.path.abspath(os.path.dirname(__file__))
```
Comment (bmerry) [inline setup.py]: Why abspath?

Comment (ludwigschwardt) [reply setup.py]: Ask @srid, the co-maintainer at [python-gnureadline](https://github.com/ludwigschwardt/python-gnureadline). I lifted it from its `setup.py` and deposited it freely everywhere :-)

Comment (ludwigschwardt) [reply setup.py]: Maybe it makes more sense the other way around, as found on various forums?

```Python
here = os.path.dirname(os.path.abspath(__file__))
```

Comment (bmerry) [reply setup.py]: Still not clear to me why you'd need the abspath at all.

Comment (bmerry) [reply setup.py]: On the other hand I guess it probably doesn't hurt, so feel free to keep it if it's going to be a pain to remove. FYI spead2 skips the abspath: https://github.com/ska-sa/spead2/blob/1e8f7d6f6ed16036f3e999ae00b8c17efba29663/setup.py#L132

**Pull request merged**